sdk/lib/cryptlib/sha1.h

```diff
@@ -11,10 +11,9 @@ extern "C" {
 /* SHA Context Structure Declaration */
 typedef struct
 {
-   ULONG Unknown[6];
-   ULONG State[5];
-   ULONG Count[2];
-   UCHAR Buffer[64];
+    UCHAR Buffer[64];
+    ULONG State[5];
+    ULONG Count[2];
```

Comment:
What about WineSync'ed `crypt_sha.c` test and other code places?
Moreover, do we need our own apitest, or could we improve existing winetests, possibly upstream?

Reply:
The latest wine code, the structure layout still incorrent, and test verify the hash result only, didn't verify the layout. So I add an own apitest.